<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Jogo</class>
 <widget class="QDialog" name="Jogo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jogo</string>
  </property>
  <widget class="QLabel" name="temporizador">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman [urw]</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tempo restante é: </string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_op">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman [urw]</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Operação: </string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>530</width>
     <height>163</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>261</width>
        <height>161</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>261</width>
        <height>161</height>
       </size>
      </property>
      <property name="text">
       <string>Iniciar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="minimumSize">
       <size>
        <width>261</width>
        <height>161</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>261</width>
        <height>161</height>
       </size>
      </property>
      <property name="text">
       <string>Voltar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>451</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="txt_resultado"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Enviar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
